psamm/lpsolver/cplex.py

```diff
@@ -246,7 +246,8 @@ def set_objective(self, expression):
         if len(quad) > 0:
             self._cp.objective.set_quadratic_coefficients(quad)
 
-        self._cp.objective.set_offset(expression.offset)
+        if self._cp.get_version() >= '12.6.2.0':
+            self._cp.objective.set_offset(expression.offset)
```

Comment:
I don't think this is a good way of comparing versions. For example,

``` python
>>> '9.8' >= '12.6.2.0'
True
```

Instead, I would just check that the method exists:

``` python
if hasattr(self._cp.objective, 'set_offset'):
    ...
```